numpy/polynomial/polynomial.py

```diff
@@ -1257,7 +1257,14 @@ def polyfit(x, y, deg, rcond=None, full=False, w=None):
         chosen so that the errors of the products ``w[i]*y[i]`` all have the
         same variance.  When using inverse-variance weighting, use
         ``w[i] = 1/sigma(y[i])``.  The default value is None.
-
+    cov : bool, optional
+        If given and not False, return the covariance matrix. By default, 
```

Comment:
Probably should be `If True, return the covariance matrix. ...`

Reply:
You are right. This is based on old documentation about polyfit